modules/custom_operations/user_ie_extensions/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# [cmake:extension]
+cmake_minimum_required(VERSION 3.13)
+project(OpenVINOExtensions)
+
+set(CMAKE_CXX_STANDARD 11)
+
+set(TARGET_NAME "user_cpu_extension")
+
+find_package(OpenVINO REQUIRED)
+find_package(OpenCV REQUIRED COMPONENTS core)
+find_package(TBB REQUIRED tbb tbbmalloc)
+
+file(GLOB_RECURSE SRC *.cpp)
+
+add_library(${TARGET_NAME} SHARED ${SRC})
+
+target_compile_definitions(${TARGET_NAME} PRIVATE IMPLEMENT_OPENVINO_EXTENSION_API)
+
+if (OpenCV_FOUND)
+  target_include_directories(${TARGET_NAME} PRIVATE ${OpenCV_INCLUDE_DIRS})
+endif()
+
+target_link_libraries(${TARGET_NAME} PRIVATE openvino::runtime ${TBB_IMPORTED_TARGETS})
```

Comment:
latest TBB versions do not provide `TBB_IMPORTED_TARGETS` variable. Let's link explicitly with targets